src/pages/CartPage/CartContext.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  PropsWithChildren,
+  createContext,
+  useContext,
+  useReducer,
+} from "react";
+import {
+  CartProduct,
+  ActionType,
+  ActionIncreaseAmountPayload,
+  ActionDecreaseAmountPayload,
+} from "./types";
+import { cartReducer } from "./cartReducer";
+
+interface createContextInterface {
```

Comment:
Use pascal case for interfaces and types
```suggestion
interface CreateContextInterface {
```